.github/ISSUE_TEMPLATE/new-feature-or-discussion.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Request a new GEOS-Chem feature or start a discussion
+---
+
+<!--- Before submitting an issue please see:
```

Comment:
I suspect most people won't do this. What about this idea, or would it be too heavy-handed?

### Review contributing guidelines

[checkbox] I have reviewed the GEOS-Chem [contributing guidelines](https://geos-chem.readthedocs.io/en/stable/reference/CONTRIBUTING.html)

Reply:
Now done in commit 581e361a